### ISSUE TITLE
Can not create new config unter Ubuntu 15.04

### ISSUE THREAD

**Issue by nextgenthemes** [openSUSE/snapper#173]
Ubuntu 15.04
snapper 0.2.4

I had this previously running on Ubunut 14.10. I had my big 4tb partition @home subvolume @home mounted as /home
And I created a seperate subvolume for the snapshots and mounted it to /home/.snapshots.

Now on my new system I have my root and /home in a SSD that is ext4 formatted and the big partition and subvolumes and mounted under /media/4tb and /media/4tb/.snapshots.

I had problems with the subvolumes filesystem/dir/something was "read.only". I tried to create and new subvolume for the snapshots and also tried to unmount the snapshot subvolume and just use the .snapshots folder but nothing worked.

There are no configs created in /etc/snapper/configs/

```
user@ /media/4tb/.snapshots                                                                                                      [0:03:13] 
> $ sudo snapper --no-dbus -v -c data create-config /media/4tb                                                                                
create subvolume failed, ioctl(BTRFS_IOC_SUBVOL_CREATE) failed, errno:17 (Die Datei existiert bereits)
Erstellung der Konfiguration fehlgeschlagen (creating btrfs snapshot failed).
```

Please try to help me.

//just updated to snapper snapper 0.2.6 (from official repo) but no luck

**Comment by aschnell** [comment]
When you create the config the .snapshots subvolume must not exist. Snapper will create that snapshot.

If it sill fails please provide /var/log/snapper.log.

**Comment by nextgenthemes** [comment]
Danke, that worked!

Still a bit confused since I remember looking that the /etc/fstab of SUSE and saw that you mount the subvolume for the snapshots seperately. I previously did this and it worked fine, I still have the subvolume and there are snapshots on it. 
```
/dev/mapper/4tb         /media/4tb             btrfs  defaults,space_cache,autodefrag,subvol=@home             0  0
/dev/mapper/4tb         /media/4tb/.snapshots  btrfs  defaults,space_cache,autodefrag,subvol=@home-snapshots   0  0
```

So I not have to do mount it and just use the subvolume that is created inside my data subvolume as a normal "folder" I guess.

**Comment by mozzbozz** [comment]
Hello,
I have exactly the same problem. However, I'm wondering how exactly it would be possible to "import" an existing config for an existing btrfs volume which was used with snapper before. Would I need to always delete the complete `.snapshot` directory after reinstalling the OS and snapper? Or is there a simpler way to just script-automated copy-paste the config to snapper and tell it "use this one"?

**Comment by nextgenthemes** [comment]
Not sure ow much this relates but since then I had a OS reinstall and I had to manually create a new (old) config. But I was unable to create a default config because the .snapshots subvolume was already there. I think snapper should provide a way to reinitialise "old" snapshots rather then failing when there already some.

**Comment by mozzbozz** [comment]
Yep, that's exactly what I meant. There is some kind of workaround (delete all old snapshots), but that doesn't really serve the purpose for which snapper is intended - but maybe we're missing something here and there is some alternative way we've missed here.

**Comment by elspru** [comment]
what you do is you move the old .snapshots to say snapshots.bak,  then create-config  then delete the .snapshots it created, and move snapshots.bak to .snapshots

**Comment by nextgenthemes** [comment]
I do not like the delete part, snapper should detect old configs and old snapshots and adapt and continue. I do not know how snapper operates, is it save to move old snapshopts in there? Because the counting starts with 0 or 1 with a "new" config. Seems very hacky.

What I did last time was renamed the snapshots folder/subvolume to .snapshots_old and then just created a new config.
